sandbox/src/main/java/org/dcsa/conformance/sandbox/ConformanceOrchestrator.java

```diff
@@ -231,27 +231,22 @@ public JsonNode handleGetPartyPrompt(String partyName) {
   public void handlePartyInput(JsonNode partyInput) {
     log.info("ConformanceOrchestrator.handlePartyInput(%s)".formatted(partyInput.toPrettyString()));
     if (currentScenarioId == null) {
-      log.info(
-          "Ignoring party input %s: no scenario is currently active"
-              .formatted(partyInput.toPrettyString()));
+      log.info("Ignoring party input: no scenario is currently active");
```

Comment:
Why don't we log the party Input now? Has it been logged anywhere else already?

Reply:
Yes indeed. No problem GJ, that happens over time, when software evolves. 

@palatsangeetha, did you see the description of this PR? I put a note in there, to help you guys review the PR: "Remove double logging of partyInput". No issues if you missed it, but just to let you know that it might help you in the future as well.

Reply:
No problem. Thanks for checking!